src/placeos/api_wrapper/endpoint.cr

```diff
@@ -88,6 +88,11 @@ module PlaceOS
         {% end %}
       {% end %}
 
+      if client.host_header
+        headers ||= HTTP::Headers.new
+        headers["Host"] = client.host_header
+      end
```

Comment:
This line is causing CI to fail https://github.com/PlaceOS/crystal-client/pull/18/checks?check_run_id=2724509156#step:5:95

```suggestion
      if host_header = client.host_header.presence
        headers ||= HTTP::Headers.new
        headers["Host"] = host_header
      end
```